crates/sargon/src/hierarchical_deterministic/bip39/bip39_word/bip39_word.rs

```diff
@@ -1,7 +1,7 @@
 use crate::prelude::*;
 
 /// A word in the BIP39 word list of `language` at known `index` (0-2047).
-#[derive(Zeroize, Clone, Debug, PartialEq, Eq, Hash, uniffi::Record)]
+#[derive(Zeroize, Clone, Debug, PartialEq, Eq, Hash)]
```

Comment:
@GhenadieVP try find replace all "Zeroize" -> "ZeroizeOnDrop, Zeroize" in the whole `sargon` crate, or otherwise manually go through all derives and try adding ZeroizeOnDrop.